diff --git a/node_modules/react-pdf-highlighter/dist/src/components/PdfHighlighter.js b/node_modules/react-pdf-highlighter/dist/src/components/PdfHighlighter.js
index d64a8a4..f035f74 100644
--- a/node_modules/react-pdf-highlighter/dist/src/components/PdfHighlighter.js
+++ b/node_modules/react-pdf-highlighter/dist/src/components/PdfHighlighter.js
@@ -271,6 +271,8 @@ class PdfHighlighter extends PureComponent {
       removePageBorders: true,
       linkService
     });
+    this.viewer.eventBus = eventBus;
+
     linkService.setDocument(pdfDocument);
     linkService.setViewer(this.viewer);
     this.viewer.setDocument(pdfDocument);
